Music Advisor/task/src/advisor/Main.java

```diff
@@ -0,0 +1,26 @@
+package advisor;
+
+import advisor.controller.Controller;
+import advisor.view.CommandLineView;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        String access = "https://accounts.spotify.com";
+        String resource = "https://api.spotify.com";
+        int pageSpace = 5;
```

Comment:
you can add `Apache Commons Lang` dependency to your project and use `defaultIfNull()` method below